### DISCUSSION TITLE
Cannot find ""Get API snippet" [Apr 30]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5610] [Discord Threads]
> 🙋‍♂️ **damianospark**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### which version are you using now?  
#  
> 🙋‍♂️ **wkw**  
#### im able to see that option with the latest version  
#  
> 🙋‍♂️ **damianospark**  
#### Since it's my first time to use nocodb, I am not aware of how to check the version. However, I just cloned the master branch yesterday.  
#  
> 🙋‍♂️ **damianospark**  
#### Node: **v12.22.6**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **true**   
#### Database: **pg**   
#### ProjectOnRootDB: **false**   
#### RootDB: **pg**   
#### PackageVersion: **0.84.10**  
#  
> 🙋‍♂️ **damianospark**  
#### this information copied from "copy project info"  
#  
> 🙋‍♂️ **wkw**  
#### the latest version is 0.105.3 tho.  
#  
> 🙋‍♂️ **wkw**  
#### did you clone from our repo?  
#  
> 🙋‍♂️ **damianospark**  
#### yes. My command I used was " git clone https://github.com/nocodb/nocodb.git "  
#  
> 🙋‍♂️ **wkw**  
#### oh wait  
#  
> 🙋‍♂️ **damianospark**  
#### Yes.  
#  
> 🙋‍♂️ **wkw**  
#### u r using docker  
#  
> 🙋‍♂️ **damianospark**  
#### Yes.  
#  
> 🙋‍♂️ **wkw**  
#### then it s not related to the clone  
#  
> 🙋‍♂️ **wkw**  
#### you can remove the local docker image n pull it again  
#  
> 🙋‍♂️ **damianospark**  
#### the compose.yml file has "image: "nocodb/nocodb:latest""  
#  
> 🙋‍♂️ **damianospark**  
#### Oh~~ I will do it and update you. Thanks a lot.  
#  
> 🙋‍♂️ **damianospark**  
#### I found the old image in my local filesystem.    
####    
#### nocodb/nocodb                                             latest                         52f07b69aaf3   13 months ago   103MB   
####    
#### removed it and "docker compose up" under docker-compose/pg folder   
####    
#### found some weird exceptions like below.  
#  
> 🙋‍♂️ **damianospark**  
#### ```   
#### nocodb-pg-nocodb-1   | GET /api/v1/db/meta/projects/ 401 29 - 1.842 ms   
#### nocodb-pg-nocodb-1   | POST /auth/token/refresh 200 305 - 3.137 ms   
#### nocodb-pg-nocodb-1   |  i [Error]: Unauthorized access   
#### nocodb-pg-nocodb-1   |     at Function.unauthorized (/usr/src/app/docker/main.js:1:35320)   
#### nocodb-pg-nocodb-1   |     at /usr/src/app/docker/main.js:1:17563   
#### nocodb-pg-nocodb-1   |     at /usr/src/app/docker/main.js:1:34623   
#### nocodb-pg-nocodb-1   |     at Layer.handle [as handle_request] (/usr/src/app/node_modules/express/lib/router/layer.js:95:5)   
#### nocodb-pg-nocodb-1   |     at next (/usr/src/app/node_modules/express/lib/router/route.js:144:13)   
#### nocodb-pg-nocodb-1   |     at t.default (/usr/src/app/docker/main.js:1:324845)   
#### nocodb-pg-nocodb-1   |     at runMicrotasks (<anonymous>)   
#### nocodb-pg-nocodb-1   |     at processTicksAndRejections (node:internal/process/task_queues:96:5)   
#### nocodb-pg-nocodb-1   | GET /api/v1/db/meta/projects/ 401 29 - 1.830 ms   
#### nocodb-pg-nocodb-1   | POST /auth/token/refresh 200 305 - 3.056 ms   
#### nocodb-pg-nocodb-1   |  i [Error]: Unauthorized access   
#### nocodb-pg-nocodb-1   |     at Function.unauthorized (/usr/src/app/docker/main.js:1:35320)   
#### nocodb-pg-nocodb-1   |     at /usr/src/app/docker/main.js:1:17563   
#### nocodb-pg-nocodb-1   |     at /usr/src/app/docker/main.js:1:34623   
#### nocodb-pg-nocodb-1   |     at Layer.handle [as handle_request] (/usr/src/app/node_modules/express/lib/router/layer.js:95:5)   
#### nocodb-pg-nocodb-1   |     at next (/usr/src/app/node_modules/express/lib/router/route.js:144:13)   
#### nocodb-pg-nocodb-1   |     at t.default (/usr/src/app/docker/main.js:1:324845)   
#### nocodb-pg-nocodb-1   |     at runMicrotasks (<anonymous>)   
#### nocodb-pg-nocodb-1   |     at processTicksAndRejections    
#### ```  
#  
> 🙋‍♂️ **damianospark**  
#### It seems nocodb reuses old data even I removed nocodb-pg-root_db-1 container.  
#  
> 🙋‍♂️ **damianospark**  
#### Oh, I found the reason. It was caused by the docker volume is not deleted and reused whenever I restart the dockers with "docker compose up". thanks for your help today.    
#### BTW. I really like your product.  
#  
> 🙋‍♂️ **wkw**  
#### when u use `docker compose down` , u can specify `-v` to delete the volume.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1091239096480972880)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
